wordpress.org/public_html/wp-content/themes/pub/wporg-support/header.php

```diff
@@ -41,36 +41,52 @@
-						<div id="primary-menu" class="menu">
-							<ul>
-								<?php
-								foreach ( $menu_items as $path => $text ) :
-									$url = parse_url( $path );
-
-									// Check both host and path (if available).
-									$is_same_host = ! empty( $url['host'] ) ? $url['host'] === $_SERVER['HTTP_HOST'] : true;
-									$is_same_path = ! empty( $url['path'] ) && false !== strpos( $_SERVER['REQUEST_URI'], $url['path'] );
-
-									$class = ( $is_same_host && $is_same_path ) ? 'class="active" ' : '';
-
-									if ( ! empty( $url['host' ] ) ) {
-										$url = esc_url( $path );
-									} else {
-										$url = esc_url( home_url( $path ) );
-									}
-								?>
-								<li class="page_item"><a <?php echo $class; ?>href="<?php echo $url; ?>"><?php echo esc_html( $text ); ?></a></li>
-								<?php endforeach; ?>
-								<li><?php get_search_form(); ?></li>
-							</ul>
+				</div><!-- .site-branding -->
+			</header><!-- #masthead -->
+		<?php elseif ( $is_forums_home ) : ?>
+			<?php echo do_blocks(
+				sprintf(
+					'<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"right":"var:preset|spacing|edge-space","left":"var:preset|spacing|edge-space"}}},"backgroundColor":"charcoal-2","className":"has-white-color has-charcoal-2-background-color has-text-color has-background has-link-color","layout":{"type":"constrained"}} -->
+					<div class="wp-block-group alignfull has-white-color has-charcoal-2-background-color has-text-color has-background has-link-color" style="padding-right:var(--wp--preset--spacing--edge-space);padding-left:var(--wp--preset--spacing--edge-space)">
+
+						<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|40","bottom":"var:preset|spacing|40"},"blockGap":"var:preset|spacing|30"}},"layout":{"type":"flex","flexWrap":"wrap","verticalAlignment":"bottom"}} -->
+						<div class="wp-block-group alignwide" style="padding-top:var(--wp--preset--spacing--40);padding-bottom:var(--wp--preset--spacing--40)">
+
+							<!-- wp:heading {"level":1,"style":{"typography":{"fontSize":"50px","fontStyle":"normal","fontWeight":"400"}},"fontFamily":"eb-garamond"} -->
+							<h1 class="wp-block-heading has-eb-garamond-font-family" style="font-size:50px;font-style:normal;font-weight:400">%s</h1>
+							<!-- /wp:heading -->
+
+							<!-- wp:paragraph {"style":{"typography":{"lineHeight":"2.3"}},"textColor":"white"} -->
+							<p class="has-white-color has-text-color" style="line-height:2.3">%s</p>
+							<!-- /wp:paragraph -->
+
 						</div>
-					</nav><!-- #site-navigation -->
-				<?php endif; ?>
-			</div><!-- .site-branding -->
-		</header><!-- #masthead -->
-		<div id="lang-guess-wrap"></div>
+						<!-- /wp:group -->
+
+					</div>
+					<!-- /wp:group -->
+
+					<div id="lang-guess-wrap"></div>
```

Comment:
Had a look into why this banner isn't working and raised a [ticket](https://dotorg.trac.wordpress.org/ticket/32).

I also discovered that https://de.wordpress.org/support/ is still live, whereas if I try to visit https://wordpress.org/support/ I end up with a web search. I expected a 301, (or worst case a 404), so this behaviour is pretty odd. I haven't covered restyling this page in this PR as I thought it was redirected, based on https://meta.trac.wordpress.org/ticket/6841. Where do you end up @ryelle?

Seems like we need to setup redirects for .org and all subdomains if the intention is for `/support/forums/` to be the homepage from now on.

Reply:
@adamwoodnz Can you document that on a meta.trac ticket instead, since the one you linked is private?

Reply:
Copied the details over to https://meta.trac.wordpress.org/ticket/7504

Reply:
The Russian version has another template again https://ru.wordpress.org/support/

> Seems like we need to setup redirects for .org and all subdomains if the intention is for `/support/forums/` to be the homepage from now on.

@WordPress/meta-design what is the intention here; do we want a consistent homepage experience for `en` and Rosetta sites like the design shows, or is a homepage at `/support/` still something that should be supported?

Reply:
> The Russian version has another template again 

Looks like this is the same template as the Japanese version, just with different widgets.

> is a homepage at /support/ still something that should be supported?

I thought that `/support/` was going to come back to the main site as well (with https://meta.trac.wordpress.org/ticket/6841)? 

If `/support/` is removed on rosetta, how would locale site visitors get to that documentation? Eventually `/documentation` will include localized content, but there's no proposal or timeline from #docs, so IMO we need to keep that info available here.

Reply:
Makes sense to bring it back to me. I think what is adding to the confusion is that the designs have dropped almost all mention of 'Support', and the link from the global nav is also 'Forums', so I'm not quite sure what a Support landing page should look like. Maybe we need to pause and allow some design time around this problem?

Reply:
I'm really not sure what to do about the widget style homepages, as the content can't be handled with our existing card styles, there are so many links.

<img src="https://github.com/WordPress/wordpress.org/assets/1017872/0784930a-e6bd-4737-91fe-35743039a3f4" width="300">

As a first step I think I'll convert the header to the new style, and leave the widgets as is. I'd appreciate your thoughts please @WordPress/meta-design

Reply:
Indeed, we need the new header style rather than the tall blue one. 

I think the content of the page can relatively easily be converted to one of the new patterns we have. I assume it's this page? https://ja.wordpress.org/support/

My first instinct here would be to remove the icons. Secondly, to explore converting that list of links into one of the existing patterns such as here, from [Learn](https://www.figma.com/file/ZKaf6u8mIHkAanluWafXAp/Learn?type=design&node-id=2149%3A5036&mode=design&t=sy7CaBe9i3xQgNph-1):

![Screenshot 2024-03-05 at 09 41 42](https://github.com/WordPress/wordpress.org/assets/1204802/9c6ace1c-bfc4-4bdd-a3fc-cc33cb663dd7)

Whether that's feasible or not, just left aligning the heading and removing the icon could be a start.

Reply:
OK thanks, sounds good. I'll look at expanding our card grid styles to handle cases where the whole card isn't clickable, with nested links.